### PR TITLE
Added Support for Restore cmdlets

### PR DESCRIPTION
**Pull request by GaryJBlake**

### CODE REVIEW

File: docs/functions/Backup and Restore/Get-VCFRestoreTask.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Get-VCFRestoreTask
+
+### Synopsis
+Gets the backup configuration of NSX Manager and SDDC Manager
```
Comment (LifeOfBrianOC) [inline docs/functions/Backup and Restore/Get-VCFRestoreTask.md:4]: Is this synopsis correct?

File: powerVCF.psm1 [hunk 2]
```diff
@@ -347,6 +369,75 @@ Function Start-VCFBackup
...
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$backupFile,
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$passphrase
+    )
+
+  	Try {
+        createCloudBuilderHeader
```
Comment (LifeOfBrianOC) [inline powerVCF.psm1:395]: Should this function be made more generic as this isnt a cloud builder use case?

Comment (GaryJBlake) [reply powerVCF.psm1:395]: Good catch I meant to go back and change that as I was juggling between the two auth methods. Will look at it now

File: docs/functions/Backup and Restore/Get-VCFRestoreTask.md (hunk 1 above)
Comment (GaryJBlake) [reply docs/functions/Backup and Restore/Get-VCFRestoreTask.md:4]: Good catch

Comment (GaryJBlake) [reply docs/functions/Backup and Restore/Get-VCFRestoreTask.md]: Fixed

**Pull request merged**